src/App.js

```diff
@@ -1,27 +1,11 @@
-import logo from './logo.svg';
+import React from 'react';
 import './App.css';
+import Greeting from './components/greeting';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Greeting />
     </div>
   );
```

Comment:
- [ ] Great job so far, however, It will be great for the `App` component to have a route that leads to the `Greeting` component. This is a major requirement for this project to be approved. ⬇️ and

![Screenshot from 2023-01-03 03-31-44](https://user-images.githubusercontent.com/79058364/210289974-c72f4748-1371-4647-93eb-885b397f96af.png)

Reply:
Well resolved 👍🏾 

![image](https://user-images.githubusercontent.com/38088818/236182265-1be6b28f-05f3-46f7-b170-ccb9a0ae552b.png)